pkgs/applications/networking/cluster/terraform/default.nix

```diff
@@ -104,8 +104,8 @@ in rec {
   terraform_0_10-full = terraform_0_10.withPlugins lib.attrValues;
 
   terraform_0_11 = pluggable (generic {
-    version = "0.11.7";
-    sha256 = "0q5gl8yn1f8fas1v68lz081k88gbmlk7f2xqlwqmh01qpqjxd42q";
+    version = "0.11.8";
+    sha256 = "1kdmx21l32vj5kvkimkx0s5mxgmgkdwlgbin4f3iqjflzip0cddh";
     patches = [ ./provider-path.patch ];
     passthru = { inherit plugins; };
   });
```

Comment:
Terraform seems to only release incrementally. Once 0.11 was out they only published TINY releases for 0.11. In general they are pretty good with backward-compatibility in TINY releases so one per MAJOR.MINOR should be ok.

After that we should limit the number of concurrent versions where N~=2 to make migrations easier. Some times it's not possible because for example `terragrunt` was depending on an old release.

Reply:
This was reported by upstream: https://github.com/hashicorp/terraform/issues/15178#issuecomment-413925056

Reply:
Thanks, good to know. I already started to see plugins complaining about protocol versions so we'll probably drop 0.11 soon.